grakn-engine/src/main/java/ai/grakn/engine/controller/response/Keyspace.java

```diff
@@ -37,45 +38,53 @@
 @AutoValue
 public abstract class Keyspace implements Jacksonisable {
 
+    @API
```

Comment:
Kind of neat how we are accidentally marking what parts of our REST API are API.